testsuite/tests/typing-misc/pr6416.ml

```diff
@@ -363,11 +363,11 @@ Error: Signature mismatch:
          type a = M/1.t
        is not included in
          type a = M/2.t
-       Line _, characters 14-42:
+       Line 2, characters 14-42:
   struct type t module M = struct type t end type a = M.t end;;
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 Definition of module M/1
-Line _:
+File "_none_", line 1:
```

Comment:
This is one instance of `Location.none` emitted by the compiler and bubbling up :).